mavproxy/source/docs/development/mavdevenvlinux.rst

```diff
@@ -13,20 +13,43 @@ The pip-installed MAVProxy will need to uninstalled (if already installed) to pr
 
 .. code:: bash
 
-    pip uninstall MAVProxy
+    python3 -m pip uninstall MAVProxy
 
 Use git to download the MAVProxy source:
 
 .. code:: bash
 
     git clone https://github.com/ArduPilot/MAVProxy.git
 
-After making the desired changes, MAVProxy is required to be installed
-(any change to the modules won't work otherwise). This needs to happen after any
-changes to the source code. This can be done by:
 
-.. code:: bash
 
-    python setup.py build install --user
+.. tabs::
+
+   .. group-tab:: User Python Environment
+
+    After making the desired changes, MAVProxy is required to be installed
+    (any change to the modules won't work otherwise). This needs to happen after any
+    changes to the source code. This can be done by:
+
+      .. code-block:: bash
+
+        python3 setup.py build install --user

```

Comment:
Both tridge and myself have switched to `python3 -m pip install .` for this step.

Reply:
Good info. Seems like `wx` is not available in the virtual environment, but that host workflow works.